src/main/java/org/apache/maven/shared/dependency/analyzer/asm/DefaultMethodVisitor.java

```diff
@@ -157,4 +160,14 @@ private void addTypeSignature(final String signature) {
             new SignatureReader(signature).acceptType(signatureVisitor);
         }
     }
+
+    @Override
+    public void visitInvokeDynamicInsn(
```

Comment:
Insn --> Instruction
I had to think about this one to figure it out.
In general, avoid abbreviations

Reply:
The naming is controlled by ASM. This class extends MethodVisitor from ASM.